.changeset/young-coins-marry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/preview-middleware': patch
+---
+
+Add Custom Table Column quick action bug fix
```

Comment:
This is very vague and it is not clear what issue was actually fixed.

Reply:
Improved